Bottom: 524a8a31657b2cd98a8e4d0a54ca0187f6ce86f6
Top:    88e5b1835918fdc8b0f0f8a0425ab0063c68e989
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-11-09 22:51:00 +0000

Refresh of try-a-new-different-sized-vbo-

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 5ed9400..436b63e 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -175,12 +175,6 @@ hidgl_flush_triangles (triangle_buffer *buffer)
       glUnmapBuffer (GL_ARRAY_BUFFER);
       buffer->triangle_array = NULL;
     } else {
-      /* NB: We only upload the portion of the buffer we've used */
-#if 0
-      glBufferSubData (GL_ARRAY_BUFFER, 0,
-                       BUFFER_STRIDE * buffer->vertex_count,
-                       buffer->triangle_array);
-#endif
       glBufferData (GL_ARRAY_BUFFER,
                     BUFFER_STRIDE * buffer->vertex_count,
                     buffer->triangle_array,
